konig-schemagen/src/main/java/io/konig/schemagen/sql/SqlTableGenerator.java

```diff
@@ -105,9 +141,16 @@ private SqlColumn column(Shape shape, PropertyConstraint p) {
 				throw new SchemaGeneratorException(message);
 			}
 			FacetedSqlDatatype datatype = datatypeMapper.type(p);
-			boolean nullable = !p.isRequiredSingleValue();
-				
-			return new SqlColumn(predicate.getLocalName(), datatype, keyType, nullable);
+			boolean nullable = !p.isRequiredSingleValue();	
+			String columnName=predicate.getLocalName();
+			if(shape.getUsesAbbreviationScheme()!=null){
```

Comment:
It is not enough to just test that the scheme is non-null.
You must use the abbreviations from that scheme and that scheme only.